frontend/src/components/Nav/MiniMenu.vue

```diff
@@ -20,15 +20,23 @@
           class="menu-item"
         >{{ item.text }}</router-link>
       </div>
-      <slot/>
+      <div
```

Comment:
Looks great but shouldn't it be a button or h3 or something with `cursor: pointer;` ? I guess this doesn't matter much given it's for mobile

Reply:
yeah because it's mobile doesn't matter.